views/index.ejs

```diff
@@ -40,7 +40,7 @@
         <a class="btn primary-button" href="/contact">Contact Us</a>
     </div>
     <div class="index-sponsor-container">
-        <h2 class="index-sponsor-text row center">Our Sponsors</h2>
+        <h2 class="index-sponsor-text row mx-auto">Our Sponsors</h2>
```

Comment:
seems like you have removed the `center` class?

Reply:
`center` class is located in `style.css` with `.center {
    text-align: center;
}
` and I don't think it is removed